lib/create-organization-api.js

```diff
@@ -74,8 +77,53 @@ export default function createOrganizationApi ({
       .then(response => wrapOrganizationMembershipCollection(http, response.data), errorHandler)
   }
 
+  /**
+   * Gets an Organization Space Membership
+   * @memberof ContentfulOrganizationAPI
+   * @param {string} id - Organiztion Space Membership ID
+   * @return {Promise<OrganizationSpaceMembership.OrganizationSpaceMembership>} Promise for a Organization Space Membership
```

Comment:
Can you change all docs to say Space Membership instead?
Also the return type declaration should be something like
```js
@return {Promise<SpaceMembership>} Promise for a Space Membership
```